app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -162,8 +162,8 @@
     <string name="Leave_the_current_channel">Sair do canal atual</string>
     <string name="Displays_action_text">Exibir texto de ação</string>
     <string name="Direct_message_someone">Enviar DM para alguém</string>
-    <string name="Mute_someone_in_room">Mutar alguém</string>
-    <string name="Unmute_someone_in_room">Desmutar alguém na sala</string>
+    <string name="Mute_someone_in_room">Silenciar alguém</string>
+    <string name="Unmute_someone_in_room">De-silenciar alguém na sala</string>
```

Comment:
I wish we had a proper antonym for "Silenciar" in PT_BR. Since we used "Mutar" and "Desmutar" before which I think does not exist, IMHO your fix is more accurate.

Reply:
The only option I'm currently seeing is to use `Remover silenciamento de alguém na sala`.
But i think this `new word` makes sense too :smiley: